front/src/modules/trainschedule/components/TimetableV2/hooks.ts

```diff
@@ -146,6 +164,7 @@ const useTrainSchedulesDetails = (
                   duration: trainSummary.time,
                   pathLength: formatKmValue(trainSummary.length, 'millimeters', 1),
                   mechanicalEnergyConsumed: jouleToKwh(trainSummary.energy_consumption, true),
+                  scheduledPointsNotHonored: trainSummary.scheduled_points_honored,
```

Comment:
I think you forgot the ! here :)
```suggestion
                  scheduledPointsNotHonored: !trainSummary.scheduled_points_honored,
```

Reply:
yes, I forgot. Done